src/main/java/net/sf/jabref/model/entry/BibtexEntry.java

```diff
@@ -60,6 +61,14 @@
     private boolean searchHit;
     private boolean groupHit;
 
+    private String serialization;
+
+    /*
+    * marks if the complete serialization, which was read from file, should be used.
+    * Is set to false, if parts of the entry change
+     */
+    private boolean useCustomSerialization;
```

Comment:
Good point, I'll do some renaming. On second thought, I will also replace the `Deque` with a `Stack`, because this is the way in which it is used.